test/sanity.js

```diff
@@ -42,8 +65,10 @@ transformFile('test/fixtures/test-multiple-svg.jsx', {
 }, (err, result) => {
   if (err) throw err;
   assertReactImport(result);
+  assertObjectAssignImport(result);
+  assertDefaultProps(false, result);
   validateDefaultProps(result);
-  console.log('test/fixtures/test-multiple-svg.jsx', result.code);
+  console.log('test/fixtures/test-multiple-svg.jsx\n', result.code);
```

Comment:
So we know, I've thought it was clearer to read the outputs with a `\n`, but if you want I can revert this